man/test.data.table.Rd

```diff
@@ -24,11 +24,4 @@ test.data.table(verbose=FALSE, pkg="pkg", silent=FALSE,
 When \code{silent} equals to \code{TRUE} it will return \code{TRUE} if all tests were successful. \code{FALSE} otherwise. If \code{silent} equals to \code{FALSE} it will return \code{TRUE} if all tests were successful. Error otherwise.
 }
 \seealso{ \code{\link{data.table}} }
-\examples{
-\dontrun{
-  library(data.table)
-  test.data.table()
```

Comment:
this was causing http://r.789695.n4.nabble.com/problematic-dev-off-after-running-examples-without-plots-td4753880.html